src/Common/src/Interop/Unix/System.Net.Security.Native/SecuritySafeHandles.cs

```diff
@@ -75,6 +133,12 @@ public void SetGssContext(SafeGssContextHandle context)
             _context = context;
         }
 
+        public void SetAuthenticationPackage(int isNtlm)
```

Comment:
because we are returning as "int"  from native layer and directly passing to this function.